applications/GeoMechanicsApplication/tests/cpp_tests/test_dgeosettlement.cpp

```diff
@@ -0,0 +1,118 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "testing/testing.h"
+#include "custom_workflows/dgeosettlement.h"
+#include "input_utility_stub.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing {
+
+const std::string parameter_json_settings = "{"
+                                            "\"solver_settings\":"
+                                            "{"
+                                            "\"model_part_name\":\"test\","
+                                            "\"model_import_settings\":"
+                                            "{"
+                                            "\"input_type\": \"mdpa\","
+                                            "\"input_filename\": \"mesh_stage1\""
+                                            "},"
+                                            "\"material_import_settings\": "
+                                            "{"
+                                            "\"materials_filename\": \"MaterialParameters1.json\""
+                                            "}"
+                                            "}"
+                                            "}"; // these have material_import settings
```

Comment:
I would use a [raw string literal](https://en.cppreference.com/w/cpp/language/string_literal) here to avoid the need to escape double quotes.

Reply:
Agreed, I started using it in the next PR and it works way simpler, so I'll also rewrite this one